db/migrate/20160910043424_add_remaining_cached_votes_fields_and_indexes.rb

```diff
@@ -0,0 +1,29 @@
+class AddRemainingCachedVotesFieldsAndIndexes < ActiveRecord::Migration
+  def up
+    add_column :cards, :cached_votes_score, :integer, default: 0
+    add_column :cards, :cached_votes_up, :integer, default: 0
+    add_column :cards, :cached_votes_down, :integer, default: 0
+    add_column :cards, :cached_weighted_score, :integer, default: 0
```

Comment:
I think omitting fields that are not used is unorthodox since it's not described in the `acts_as_votable` docs anywhere. It's not going to cost much.